docs/API.md

```diff
@@ -0,0 +1,203 @@
+# ICNS API Documentation
+
+Introduction
+------------
+
+Welcome to the ICNS API documentation! The ICNS API allows you to interact with the Interchain Nameservice, a decentralized protocol that enables users to register and manage human-readable names on the Interchain.
+
+Contract Address
+----------------
+
+*   Name NFT
+    *   Mainnet: TBD
```

Comment:
```suggestion
    *   Mainnet:  `osmo1xk0s8xgktn9x5vwcgtjdxqzadg88fgn33p8u9cnpdxwemvxscvast52cdd`
```

Reply:
Sunny got the Name NFT & Resolver contracts flip-flopped

![image](https://user-images.githubusercontent.com/38844339/215022796-e2730892-d79c-4736-8b68-6ea8393fe2d1.png)

```suggestion
    *   Mainnet: `osmo1mypljhatv0prfr9cjzzvamxdf2ctg34xkt50sudxads9zhqnyneqjuvy26`
```